pkgs/by-name/tl/tlmi-auth/package.nix

```diff
@@ -6,17 +6,14 @@
 , ninja
 , openssl
 }:
-let
-  name = "tlmi-auth";
+
+stdenv.mkDerivation rec {
```

Comment:
Can you remove the `rec` antipattern here and instead either move the version into a `let ... in` clause or use use `mkDerivation (finalAttrs: { ... })`?